src/components/Block/Network.vue

```diff
@@ -12,15 +12,20 @@ function getLogoUrl(key) {
 
 <template>
   <Block class="hover-border">
-    <div class="flex items-center mb-1">
+    <div class="flex items-start mb-3">
       <UiAvatar
-        class="mr-2 mb-2"
+        class="mr-2"
         :imgsrc="getLogoUrl(network.key)"
         :seed="network.key"
         size="28"
       />
-      <h3 v-text="network.name" />
-      <div v-text="network.key" class="ml-1 text-color" />
+      <div class="max-w-[85%]">
```

Comment:
That percentage width made me a bit suspicious. I assume you didn't see the "..." without it? That's because that parent div container does otherwise not have any width constraints and thus no overflow is triggered. The child element, which is meant to be truncated, is just rendered off-screen.

```suggestion
      <div class="overflow-hidden">
```

This has the same effect, without limiting the width to an arbitrary 85%.